setupbase.py

```diff
@@ -64,6 +64,14 @@ def run(self):
         if not osp.isdir(COMPONENTS):
             log.info("running [yarn install]")
             run(['yarn', 'install'], cwd=repo_root)
+            log.info("installing webpack")
+            run(['npm', 'i', '-D', 'webpack'], cwd=repo_root)
+            run(['npm', 'i', 'webpack-cli', 'html-webpack-plugin', 'xterm',
+                 'xterm-addon-attach', 'xterm-addon-search',
+                 'xterm-addon-web-links', 'xterm-addon-webgl',
+                 'xterm-addon-fit@0.1.0-beta2'], cwd=repo_root)
```

Comment:
Why do we have to reinstall all these packages with npm if they are installed by yarn too?

Reply:
npn installs the dependencies for the webpack to build the final version of the plugin, while yarn installs the static dependencies inside the server.